LFP_analysis/Laser_LFP_Parse_Final.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jul  6 10:06:32 2017
+
+@author: Bradly
+"""
+#Import necessary tools
+import numpy as np
+import easygui
+import tables
+import os
+import matplotlib.pyplot as plt
+from scipy import signal
+from scipy.signal import periodogram
+
+#Get name of directory where the data files and hdf5 file sits, and change to that directory for processing
+dir_name = easygui.diropenbox()
+os.chdir(dir_name)
+
+#Look for the hdf5 file in the directory
+file_list = os.listdir('./')
+hdf5_name = ''
+for files in file_list:
+	if files[-2:] == 'h5':
+		hdf5_name = files
+
+#Open the hdf5 file and create list of child paths
+hf5 = tables.open_file(hdf5_name, 'r+')
+lfp_nodes = hf5.list_nodes('/Parsed_LFP')
+laser_nodes = hf5.list_nodes('/spike_trains')
+
+# Create a LFP_laser group in the hdf5 file
+try:
+    hf5.remove_node('/LFP_Lasers', recursive = True)
+except:
+    pass
+hf5.create_group('/', 'LFP_Lasers')
+
+#Establish number of tastes and laser_combination variables
+taste_count = len(laser_nodes)
+laser_comb_count = len(laser_nodes)

```

Comment:
len(laser_nodes) doesn't give you the count of laser combinations. Your code here will deduce that the number of laser combinations is the same as the number of tastes, which isn't the case generally. The laser combinations are contained in:
```
hf5.root.ancillary_analysis.laser_combination_d_l
```
Look that up!